unirec/model/modules.py

```diff
@@ -430,3 +431,280 @@ def forward(self, hidden_states, attention_mask, output_all_encoded_layers=True)
         if not output_all_encoded_layers:
             all_encoder_layers.append(hidden_states)
         return all_encoder_layers
+
+
+# Ada-Ranker
+class NeuProcessEncoder(nn.Module):
```

Comment:
it is suggested to add some annotations for the new modules.

Reply:
Got it!